lib/export-contact-history-s3.js

```diff
@@ -36,7 +36,7 @@ function exportContactHistory(scope) {
     //Add the bucket name to the lambda's environment variables
     exportContactHistory.addEnvironment('s3bucket', contactHistoryBucket.bucketName);
     //Allow lambda to access Honeycode workbook
-    //exportContactHistory.role.addManagedPolicy(iam.ManagedPolicy.fromAwsManagedPolicyName('AmazonHoneycodeFullAccess'));
+    //exportContactHistory.role.addManagedPolicy(iam.ManagedPolicy.fromAwsManagedPolicyName('AmazonHoneycodeWorkbookFullAccess'));
```

Comment:
Will this be an alternative to use selective workbookId for the policy? What are we trying to show with this commented line?

Reply:
Added a comment explaining the difference between the two options and recommended using the more specific policy